src/App.tsx

```diff
@@ -1,25 +1,137 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { FullTodo, Todo } from './types/Todo';
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+const prepareTodos = (
+  users: User[],
+  todos: Todo[],
+): FullTodo[] => {
+  return todos.map((todo) => ({
+    ...todo,
+    user: users.find((user) => user.id === todo.userId) || null,
+  }));
+};
+
+const prepareTodo = (
+  currentUser: User,
+  title: string,
+  todos: Todo[],
+): FullTodo => {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return ({
+    id: maxId + 1,
+    title,
+    completed: false,
+    userId: currentUser.id,
+    user: currentUser,
+  });
+};
+
+const visibleTodos = prepareTodos(usersFromServer, todosFromServer);
+const users = usersFromServer;
+
+export const App: React.FC = () => {
+  const [userSelected, setUserSelected] = useState('0');
```

Comment:
in my opinion it's more appropriate to set to `value` not `user.name` but `user.id`